implementingSwashbuckle.md

```diff
@@ -0,0 +1,133 @@
+
+Reading through the [DapperDox documentation](http://dapperdox.io/docs/spec-resource-definitions), I discovered that "When specifying a resource schema object...DapperDox requires that the optional schema object title member is present." This requirement was problematic, because Swashbuckle does not include a method for adding members to a schema in the generated swagger.json file. Additionally, it took some tinkering in the code for me to realize that the missing title member on the ```definitions``` model is what caused DapperDox to break.
+
+## Fixing the output
+
+The Swashbuckle documentation offered little help in this regard, so I turned to one of our developers. After reviewing the code together, my developer counterpart created a ```SwaggerTitleFilter``` method that adds a title member to the definitions model in the resulting swagger.json file. The title member displays in the generated documentation as a link to the referenced object, creating a hyperlink between the two objects.
+
+The following code implements an IModelFilter that causes Swashbuckle to generate a title member for any schema.
+
+```
+namespace MyAPI.Swagger
+{
+    public class SwaggerTitleFilter : IModelFilter
+    {
+        public void Apply(Schema schema,  ModelFilterContext mfc)
+        {
+            schema.vendorExtensions.Add("title", mfc.SystemType.Name);
+        }
+    }
+}
+```
+I compiled the code and Swashbuckle generated an updated swagger.json file. With the title member added to the swagger.json output, I pointed DapperDox at the directory containing my swagger.json file.
+
+```
+.\dapperdox -spec-dir=C:\Bitbucket\APIproject\source
+```
+
+I opened a browser and entered ```http://localhost:3123```, which is where DapperDox runs by default, and it worked! DapperDox displayed my swagger.json file and created interactive documentation that clearly displays the requests, responses, and query parameters for the API.
+
+## Next steps
+
+With this framework in place, we can extend Swashbuckle to future APIs and use DapperDox to host the swagger.json file for each. The resulting output is a swagger.json file that lives with the code, and documentation that developers and the support engineers can access locally by running a single command.
+
+Each time that the code builds, the swagger.json file updates with the most current information. Developers and support engineers just run the ```.\dapperdox``` command and specify the directory where the swagger.json file lives. As the code changes, so does the documentation, so technical debt approaches zero.
+
+To add documentation beyond just the generated JSON output, DapperDox worked incredibly well. The pain of determining why DapperDox was broken and the additional coding required to fix the problem was worth the effort, and we are poised to integrate this process into the next APIs that are developed.
```

Comment:
Oooo please say more about adding documentation beyond the generated JSON output! Everyone struggles with this at some point while documenting APIs.